src/Umbraco.Abstractions/CompositionExtensions.cs

```diff
@@ -22,6 +23,15 @@ public static ComponentCollectionBuilder Components(this Composition composition
         public static DashboardCollectionBuilder Dashboards(this Composition composition)
             => composition.WithCollectionBuilder<DashboardCollectionBuilder>();
 
+
+        /// <summary>
+        /// Gets the content finders collection builder.
+        /// </summary>
+        /// <param name="composition">The composition.</param>
+        /// <returns></returns>
+        public static DataEditorWithMediaPathCollectionBuilder DataEditorsWithMediaPath(this Composition composition)
```

Comment:
Will think of a different name :) I'll push something shortly and let you know

Reply:
I have pushed a new rev: 13a56eb08549f196eeb8039b656c24cf91c452aa which renames and refactors this a little to simplify this and decouple it from property editors since the logic was still relying on property editors having this interface which isn't what we want.